.github/workflows/build-ppmseq-docker-for-apl.yml

```diff
@@ -0,0 +1,75 @@
+name: ppmseq
+run-name: "Build PPMSeq Docker Image"
+
+on:
+  workflow_dispatch:
+
+permissions:
+  id-token: write # Required for assuming an AWS role
+  contents: read # Required for actions/checkout
+
+env:
+  AWS_REGION: us-east-1
+  AWS_ROLE: arn:aws:iam::${{ secrets.AWS_ACCOUNT }}:role/github-actions-aws-role
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ env.AWS_ROLE }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Update build status
+        if: inputs.id != ''
+        run: |
+          EXP=`date -d '+3 hours' +%s`
+          aws dynamodb put-item --table BuildStatusTable --item '{"BuildId": {"S": "${{ inputs.id }}-${{ github.repository }}"}, "commit": {"S": "${{ github.sha }}"}, "run_id": {"S": "${{ github.run_id }}"}, "ExpirationTime": {"N": "'$EXP'"}}'
```

Comment:
since this workflow is not called by the APL build job, these lines can be removed